src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -93,6 +94,14 @@ private VariantOverlapAnnotator initializeOverlapAnnotator(final FeatureInput<Va
 
         return new VariantOverlapAnnotator(dbSNPInput, overlaps);
     }
+    public static VariantAnnotatorEngine ofSelectedMinusExcluded(final VariantAnnotationArgumentCollection argumentColleciton,
```

Comment:
Add javadoc for this public method, and move it next to the other overload of `ofSelectedMinusExcluded()` in the source file.